File: software/examples/php/ExampleSimple.php
<?php

require_once('Tinkerforge/IPConnection.php');
require_once('Tinkerforge/TNGDO8.php');

use Tinkerforge\IPConnection;
use Tinkerforge\TNGDO8;

const HOST = 'localhost';
const PORT = 4223;
const UID = 'XYZ'; // Change XYZ to the UID of your TNG DO8

$ipcon = new IPConnection(); // Create IP connection
$do8 = new TNGDO8(UID, $ipcon); // Create device object

$ipcon->connect(HOST, PORT); // Connect to brickd
// Don't use device before ipcon is connected

$do8->setValue(array(TRUE, FALSE, TRUE, FALSE, TRUE, FALSE, TRUE, FALSE));

echo "Press key to exit\n";
fgetc(fopen('php://stdin', 'r'));
$ipcon->disconnect();

?>

File: software/examples/python/example_simple.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

HOST = "localhost"
PORT = 4223
UID = "XYZ" # Change XYZ to the UID of your TNG DO8

from tinkerforge.ip_connection import IPConnection
from tinkerforge.tng_do8 import TNGDO8

if __name__ == "__main__":
    ipcon = IPConnection() # Create IP connection
    do8 = TNGDO8(UID, ipcon) # Create device object

    ipcon.connect(HOST, PORT) # Connect to brickd
    # Don't use device before ipcon is connected

    do8.set_value(0, [True, False, True, False, True, False, True, False])

    input("Press key to exit\n") # Use raw_input() in Python 2
    ipcon.disconnect()
